core/templates/dev/head/domain/objects/NumberWithUnitsObjectFactorySpec.js

```diff
@@ -65,44 +71,48 @@ describe('NumberWithUnitsObjectFactory', function() {
     });
 
     it('should convert itself to a string', function() {
```

Comment:
What is `itself`? Maybe more descriptive message?

Reply:
Done!